package BibleStudy;

message VerseBuffer
{
    required string book = 1;
    required int32 chapter = 2;
    required int32 verse = 3;
    required string text = 4;
    required int32 unique_id = 5;
}

message TranslationBuffer
{
    required string long_name = 1;
    required string short_name = 2;
    repeated VerseBuffer verse = 3;
}